server/src/test/java/org/opensearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -107,7 +107,7 @@
 public class CoordinatorTests extends AbstractCoordinatorTestCase {
```

Comment:
https://github.com/opensearch-project/OpenSearch/blob/a87cede9b88976e0a043e7ea8670d3de2f400c1e/server/src/test/java/org/opensearch/cluster/coordination/CoordinatorTests.java#L1099

Reply:
https://github.com/opensearch-project/OpenSearch/blob/a87cede9b88976e0a043e7ea8670d3de2f400c1e/server/src/test/java/org/opensearch/cluster/coordination/CoordinatorTests.java#L1124

Reply:
https://github.com/opensearch-project/OpenSearch/blob/a87cede9b88976e0a043e7ea8670d3de2f400c1e/server/src/test/java/org/opensearch/cluster/coordination/CoordinatorTests.java#L1140

Reply:
Comment here: https://github.com/opensearch-project/OpenSearch/blob/a87cede9b88976e0a043e7ea8670d3de2f400c1e/server/src/test/java/org/opensearch/cluster/coordination/CoordinatorTests.java#L1095
`NO_MASTER_BLOCK` -> `NO_CLUSTER_MANAGER_BLOCK`

Reply:
Good catch! Thanks for pointing out. I renamed the test method names in the new commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/704f00c14d8cec6a80cece316dc9afdc274b736b.

For the `NO_MASTER_BLOCK` in comment, could it be renamed along with the definition of `NO_MASTER_BLOCK`? `NO_MASTER_BLOCK` is a public constant, and I will rename it in the next major version (3.0).